README.md

```diff
@@ -58,6 +58,7 @@ An implementation of [failpoints][failpoint] for Golang.
     - `func Goto(label string) {}`
     - `func Continue(label ...string) {}`
     - `func Fallthrough() {}`
+    - `func Return(results ...interface{}) {}`
```

Comment:
I suppose this is introduced just for completeness?

Reply:
```go
func foo() (int, string) {
	// Some logic
	failpoint.Inject("failpoint-name", func(val failpoint.Value) (int, string) {
		// return early
		if val.(int) % 10 == 0 {
			return val.(int), "string from failpoint"
		}
		// We must write a return statement here to make the program legal
		return 0, "I do not want return anything here"
	})
	// Other logics
}
```

If we need an `early return` in failpoint with some conditions like above case. We will get unexpected result because there is always `early return` if failpoint is active. So we need a `Return` marker function.

```go
func foo() (int, string) {
	// Some logic
	failpoint.Inject("failpoint-name", func(val failpoint.Value) {
		// return early
		if val.(int) % 10 == 0 {
			failpoint.Return(val.(int), "string from failpoint")
		}
	})
	// Other logics
}
```